docs/docs/community/release-process.md

```diff
@@ -0,0 +1,68 @@
+
+## Release Captain responsibilities
+
+For every release, there is an assigned "Release Captain". The Release Captain's responsibilities are:
+
+- Manage both the release and testing processes and update the checklists as needed.
+- Communicate the status of the release to the rest of the Nebari development team and the community (through updating the checklists and adding status updates as comments).
+- Assign owners to checklist items (if not owned by the Release Captain).
+- Adjust the schedule, particularly the publishing dates, based on defects found, fixes made, holidays, vacations, and so on.
+- Ensure a new branch and milestone is created after completing a release cycle.
+
+## CalVer details
+
+Nebari releases should follow the following CalVer versioning style:
+
+```
+YYYY-MM-releaseNumber
+```
+
+:::info
+`YYYY` represents the year - i.e. `2023`
+
+`MM` represents the _non-zero padded_ month - i.e. `1` for January, `12` for December
+
+`releaseNumber` represents the current release for that month, starting at `1`. Anything above `1` represents a [hotfix (patch) release](#hotfixes).
+:::
+
+:::caution
+For the release tag, there should be NO prepended `v`
+:::
+
+For example, the first Nebari CalVer release was `2022.10.1`. If a hotfix release was needed in the same month, we increment the `releaseNumber` by 1, which would be `2022.10.2` (_this is to illustrate how the increment works, this release does not exist._)
+
+## Gitflow and branching strategy
+
+Gitflow is framework for managing `git` branches by assigning certain roles to particular branches.
+
+- `develop` - Represents the active development branch and is the _default_ branch on the GitHub repository.
+
+- `main` - Represents a production-ready state of the code-base, with an appropriate tag to match the most recent release.
+
+- `release/YYYY-MM-releaseNumber` - Represents the branch for the upcoming release.
+
+### Process
+
+Although this process is captured in the [release checklist template](https://github.com/nebari-dev/nebari/issues/new?assignees=&labels=type%3A+release+%F0%9F%8F%B7&template=release-checklist.md&title=%5BRELEASE%5D+%3Cversion%3E), it's worth making clear how branches are managed.
+
+- Active development occurs against the `develop` branch.
+- When it's time for a release, the Release Captain will create the release branch `release/YYYY-MM-releaseNumber` and preparing the branch for the release. At times, this might mean cherry-picking a commits that are needed for this release and at other times, this might mean merging `develop` into this release branch.
```

Comment:
```suggestion
- When it's time for a release, the Release Captain will create the release branch `release/YYYY-MM-releaseNumber` and prepare the branch for the release. At times, this might mean cherry-picking commits that are needed for this release and at other times, this might mean merging `develop` into this release branch.
```